ModelBundle/Repository/NodeRepository.php

```diff
@@ -733,6 +751,28 @@ public function findByParentAndRoutePattern($parentId, $routePattern, $nodeId, $
     }
 
     /**
+     * @param string $parentId
+     * @param int    $order
+     * @param string $nodeId
+     * @param string $siteId
+     *
+     * @return array
+     */
+    public function findByParentAndOrder($parentId, $order, $nodeId, $siteId)
```

Comment:
I would add `AndNotNode` in the method name to say that you will find something differente